schemas/devfile.json

```diff
@@ -638,6 +638,36 @@
       },
       "type": "array"
     },
+    "events": {
+      "description": "Bindings of commands to events",
```

Comment:
This description could probably be improved -- it's not clear that items in the array should be command names.

Reply:
Fixed